crawl-ref/source/mon-pick-data.h

```diff
@@ -393,21 +393,59 @@ static const vector<pop_entry> population[] =
 },
 
 { // Spider Nest
-  { -1,  4,  260, RISE, MONS_HORNET },
-  { -1,  6,   89, SEMI, MONS_BOULDER_BEETLE },
-  { -1,  9,  765, SEMI, MONS_DEMONIC_CRAWLER },
-  {  1,  4,  955, FLAT, MONS_WOLF_SPIDER },
-  {  1,  4,  985, FLAT, MONS_JUMPING_SPIDER },
-  {  1,  4,  910, FLAT, MONS_TARANTELLA },
-  {  1,  4, 1045, FLAT, MONS_REDBACK },
-  {  1,  4,  550, FLAT, MONS_ORB_SPIDER },
-  {  1,  6,  100, FLAT, MONS_TORPOR_SNAIL },
-  {  1,  6,  190, SEMI, MONS_EMPEROR_SCORPION },
-  {  1,  6,  155, SEMI, MONS_MOTH_OF_WRATH },
-  {  1,  9,  300, SEMI, MONS_ENTROPY_WEAVER },
-  {  1,  9,  200, PEAK, MONS_GHOST_MOTH },
-  {  2,  4,   12, FLAT, MONS_MELIAI },
-  {  2, 10,  260, PEAK, MONS_SPARK_WASP },
+  // Easy enemies:
+  {  1,  4,   200, FLAT, MONS_BOULDER_BEETLE },
+  {  1,  4,   100, FALL, MONS_BOULDER_BEETLE },
+  {  1,  4,   800, FLAT, MONS_REDBACK },
+  {  1,  4,   800, FALL, MONS_REDBACK },
+  {  1,  4,   500, FLAT, MONS_JUMPING_SPIDER },
+  {  1,  4,   900, FALL, MONS_JUMPING_SPIDER },
+  {  1,  4,   600, FLAT, MONS_TARANTELLA },
+  {  1,  4,   500, FALL, MONS_TARANTELLA },
+  {  1,  4,   400, FLAT, MONS_CULICIVORA },
+  {  1,  4,   500, FALL, MONS_CULICIVORA },
+  {  1,  4,   700, FLAT, MONS_DEMONIC_CRAWLER },
+  {  1,  4,   300, FALL, MONS_DEMONIC_CRAWLER },
+  // Harder enemies:
+  {  1,  4,   600, FLAT, MONS_WOLF_SPIDER },
+  {  1,  4,   400, RISE, MONS_WOLF_SPIDER },
+  {  1,  4,   400, FLAT, MONS_TORPOR_SNAIL },
+  {  1,  4,   200, RISE, MONS_TORPOR_SNAIL },
```

Comment:
Even with friends, I'm not sure that torpor snails are above the bottom-tier of threats by Spider. I'd vaguely lean toward swapping them with tarantellas in the tier listing. Interested in your thoughts, though.